esti/s3_gateway_test.go

```diff
@@ -328,6 +328,89 @@ func TestS3HeadBucket(t *testing.T) {
 	})
 }
 
+func TestS3CopyObjectMultipart(t *testing.T) {
+	const minDataContentLengthForMultipart = 5 << 20
```

Comment:
nit: it's a bit confusing that `minDataContentLengthForMultipart` and `largeDataContentLength` exist in different files and only used here